src/environmentd/tests/auth.rs

```diff
@@ -645,7 +645,11 @@ fn start_mzcloud(
         roles,
         role_updates_rx: Arc::new(Mutex::new(role_updates_rx)),
         now,
-        expires_in_secs,
+        expires_in_secs: if cfg!(coverage) {
+            expires_in_secs * 2
+        } else {
+            expires_in_secs
+        },
```

Comment:
This seems sketchy! I don't think this function should be manipulating `expires_in_secs`. It makes it hard for me to understand whether this test is testing the right thing. I think instead we should be bumping `const EXPIRES_IN_SECS` and `const REFRESH_BEFORE_SECS`?

I might be off on this; would be good to have @mjibson take a look. This is an extremely load bearing test so I don't want us to get the logic wrong.

Reply:
I agree with this. This fn is called in 3 places. 1 of them should be unchanged (the one hard coded to 1_000). The others sould double those both those consts. Based on the pgwire test advice below, seems like we should unconditionally double them.